apps/web/src/pages/agent.tsx

```diff
@@ -1,5 +1,5 @@
-import { AgentManagerProvider } from "~core/providers/agent-manager"
-import { WebVMProvider } from "~core/providers/web-vm"
+import { AgentManagerProvider } from "~features/agent/agent-manager-provider"
```

Comment:
can we put agent and vm inside a `demos` folder, so that the demos are namespaced and grouped together?

Reply:
Done 👍